src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <section className="calculator">
+    <div className="screen">0</div>
+    <div>AC</div>
+    <div>+/-</div>
+    <div>%</div>
+    <div className="function">/</div>
+    <div>7</div>
+    <div>8</div>
+    <div>9</div>
+    <div className="function">x</div>
+    <div>4</div>
+    <div>5</div>
+    <div>6</div>
+    <div className="function">-</div>
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
+    <div className="function">+</div>
+    <div className="zero">0</div>
+    <div>.</div>
+    <div className="function">=</div>
+  </section>
+);
```

Comment:
- [ ] According to the project requirement `Calculator.js` should have at least 1 child component. Please implement it.